docs/connecting-data/connecting-to-apis-and-external-data/datadog-api.md

```diff
@@ -2,6 +2,142 @@
 description: How to Connect DataDistillr to the Datadog API
 ---
 
-# Datadog API
+# Connecting to Datadog
 
-Details coming soon!
\ No newline at end of file
+## First Steps with Affinity
+Set up an account with [Datadog](https://www.datadoghq.com/){target=_blank}.
+
+???+ cost "Costs"
+
+    This is a paid API. [Checkout Datadog's](https://www.datadoghq.com/pricing/){target=_blank} pricing.
+
+
+???+ rlimit "Rate Limits"
+
+    All of the API endpoints are [rate limited](https://docs.datadoghq.com/api/latest/rate-limits/){target=_blank}. Once you exceed a certain number of requests in a specific period, Datadog returns an error.
+
+## How to Connect DataDistillr to Datadog
+To set up a data source connection for Datadog, you will need to have:
+
+- A unique [name](#name) for your data source connection to be used in queries.
+- The [API Key](#steps-for-getting-your-api-key) generated by Datadog.
+- The [Application Key](#steps-for-getting-your-application-key) generated by Datadog.
+
+
+
+### Data Source Form
+
+To locate the Datadog form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.&#x20;
+
+![Select API from the available choices][image-5]
+
+On the API screen, select Datadog from list of API forms as shown in the image below.
+
+![Select Datadog API from available choices][image-6]
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the Datadog API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+![Datadog Form][image-1]
+
+### Name
+
+Enter any name that will help you recognize this data source from within your query window. &#x20;
+
+!!! info "Acceptable characters include"
+
+    * lowercase alphanumeric characters
+    * underscores
+
+#### Steps for getting your Application Key
```

Comment:
Change the order of this and API key so it matches the order on the form. Also modify title to be at same level as name (###) and just name it 'Application key'